test/envtest/konnect_entities_kongcacertificate_test.go

```diff
@@ -53,7 +55,8 @@ func TestKongCACertificate(t *testing.T) {
 	t.Log("Setting up SDK expectations on KongCACertificate creation")
 	sdk.CACertificatesSDK.EXPECT().CreateCaCertificate(mock.Anything, cp.GetKonnectStatus().GetKonnectID(),
 		mock.MatchedBy(func(input sdkkonnectcomp.CACertificateInput) bool {
-			return input.Cert == deploy.TestValidCACertPEM
+			return input.Cert == deploy.TestValidCACertPEM &&
+				slices.Contains(input.Tags, "tag1")
```

Comment:
Please consider extracting `tag1` and `xconflictx` tags as constants to reduce duplication.